main/REPL/board.md

```diff
@@ -0,0 +1,95 @@
+# The Agoric Board
+
+## Introduction
+
+The Board is a shared, on-chain location where users post an object as a value and make
+it accessible to others. When a user posts a value, they receive a unique ID 
+for that value. Others can get the value just by knowing the ID. You can make 
+an ID known by any communication method; private email, a DM or other private 
+message, an email blast to a mailing list or many individuals, listing it on a website, etc.
+
+In particular, the Board is frequently used to give others access to deposit
+facets. After you post a deposit facet object to the Board, you distribute its
+Board ID string as widely as you'd like. Anyone who has that ID can use it to
+get access to its value, the deposit facet object. They can then safely deposit
+assets into the facet's associated purse without being able to withdraw assets
+from the purse or check its balance.
+
+The `ids()` method returns all of the Board's currently used IDs. 
+This means anyone can access any Board-stored value. The Board is public, 
+not private.
+
+Note that when calling from the REPL's `home` object, you must use 
+the [`E` syntax](/distributed-programming.md#communicating-with-remote-objects-using-e)
+as shown below.
```

Comment:
"Must" is too strong, as `~.` is an alternative.  Don't we describe and recommend this syntax in a few places?

Reply:
In general I've steered away from using ~. in the docs. As I recall, at least when I was starting, there were still some issues about it externally (perhaps general availability?). If there's no problem with it externally at this point, I should add something about it to the Distributed Programming doc.

Reply:
It's very useful in the REPL. Hard to use elsewhere. It could get public visibility if some of us do demos in which we have to interact with the REPL.

I sure hope we're going to have the underlying issues fixed before beta, but I don't know enough to be sure of that.

Reply:
For now then, unless you've got a strong objection, I'm going to continue to not mention ~ and thus just leave the "must" as it's appropriate if you don't know ~. I think tossing it in as alternative, but only in the REPL, would be confusing.